### PR TITLE
Update the Incident Management Process to match the recent additions

### PR DESCRIPTION
**Pull request by cattias**
New definition of Problem Investigation Manager
Added process for PMR and PMR AI follow up

### CODE REVIEW

**Comment by cattias** [comment]
https://source.redhat.com/groups/public/service-delivery/service_delivery_wiki/incident_management_process

**Comment by cattias** [comment]
@jeremyeder can you review and see if it can be merged ?
thanks

**Review by schwesig — COMMENTED** [review]
/lgtm

File: process/incident_management.md [hunk 1]
```diff
@@ -190,6 +190,17 @@ The customer support lead is the person from our first level support organizatio
 
 When an incident is initially triggered or managed internally to SRE, meaning that it was not initiated by a customer request through Customer support and when this incident has an impact to the customer then the Incident Commander needs to ensure that the right representation from Customer Support is available in the bridge.
 
+### Problem Investigation Manager
+* The Problem Investigation Manager is the SRE Manager supporting the team who is working on resolving the incident from a management perspective.
+* The Problem Investigation Manager is accountable for following up on the Post Mortem and all the action items that have been derived from it until they are all completed.
+* The Problem Investigation Manager must ensure that the PMR and all its action items are properly tracked and will be acted upon in a timely fashion by the different teams impacted according to their priorities.
+* This is a coordination and project management type of activity which requires reaching out to different external teams and potentially escalating * issues, that's the main reason why this is under the responsibility of a Manager.
```
Comment (mmazur) [inline process/incident_management.md:197]: > potentially escalating * issues

Why the * ?

File: process/incident_management.md [hunk 2]
```diff
@@ -190,6 +190,17 @@ The customer support lead is the person from our first level support organizatio
...
+* Usually, the Problem Investigation Manager will be one of the Regional Managers where the incident first occurs, whoever is available at the time. 
+* The Problem Investigation Manager doesn't need to be handed over as a follow-the-sun role, it can stay within the Region of origin.
+* If the incident ownership is fully transferred to a different region or a different team, the Problem Investigation Manager role can be handed over.
+
+** Note: If an incident is caused by the SRE that impacts the customer's workloads, the Problem Investigation Manager to immediately inform BU about the same. **
```
Comment (mmazur) [inline process/incident_management.md]:
```suggestion
** Note: If an incident is caused by the SRE and impacts the customer's workloads, the Problem Investigation Manager must immediately inform the Business Unit about it. **
```

**Review by mmazur — COMMENTED** [review]
The Investigation Manager part seems mergable. The PMR bit is very RH-specific and I need to double check what's the plan on the type of content in this repo, before I'm comfortable merging it.

File: process/incident_management.md (hunk 1 above)
Comment (cattias) [reply process/incident_management.md:197]: to make a list

**Comment by cattias** [comment]
@mmazur, I updated the PR

**Comment by cattias** [comment]
@mmazur ok to lgtm and approve too ?

**Pull request merged**